ansible/idr-omero-readonly.yml

```diff
@@ -79,16 +79,13 @@
       recurse: yes
       state: directory
 
-  # TODO: Rendering in web is current broken with omero.cluster.read_only true
   - name: OMERO.server read-only configuration
     become: yes
     copy:
       content: |
         config set -- omero.data.dir /data/OMERO
-        #config set -- omero.cluster.read_only true
-        config set -- omero.cluster.node_provider ome.security.basic.InMemoryNodeProvider
-        config set -- omero.sessions.session_manager ome.services.sessions.InMemorySessionManagerImpl
-        config set -- omero.security.event_provider ome.security.basic.InMemoryEventProvider
+        config set -- omero.cluster.read_only true
+        config set -- omero.pixeldata.memoizer.dir.local /tmp/BioFormatsCache
```

Comment:
Does this mean the NFS share https://github.com/IDR/deployment/blob/IDR-0.4.7-1/ansible/idr-omero-readonly.yml#L24 can either be removed or made read-only?

Reply:
It _could_ be made read-only, but this is actually one of the places where I think we've gone too far. Specifically, I *want* to be able to write to the global BioFormatsCache from the read-only servers. I don't need to _yet_ since we're not bumping our Bio-Formats version, but we will need to _soon_.

Reply:
@mtbc suggests in https://trello.com/c/q5L39XPZ/5-bioformatscache-is-expecting-read-write that if I omit this line, then the old behavior will be used. Likely worth testing.

Reply:
Pushed a removal of `omero.pixeldata.memoizer.dir.local`